src/phoenix/types/queue.go

```diff
@@ -1,3 +1,5 @@
+// TODO: Either add the MIT Licensing for this open-source code or use the github collection directly
```

Comment:
What should we do here?

Reply:
it seems like it works without any changes, we can use the collection directly

Reply:
Done